gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/06_add_scoring_table.yml

```diff
@@ -11,7 +11,6 @@ databaseChangeLog:
               - column: { name: api_id, type: nvarchar(64), constraints: { nullable: false } }
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.